tests/tests/reduced-permissions.spec.js

```diff
@@ -0,0 +1,255 @@
+/// <reference types="cypress" />
+
+import config from '../config';
+
+const id = Math.random()
+  .toString()
+  .substr(2, 8);
+
+const SA_NAME = 'test-sa-' + id;
+const CR_NAME = 'test-cr-' + id;
+const CRB_NAME = 'test-crb-' + id;
+
+function chooseComboboxOption(selector, optionText) {
```

Comment:
Could we move this function to a shared file? This might be useful in other tests